[DTM] - Improvements for user documentation (#1696)
SoftwareAG/c8y-docs: content/dtm/asset-types-bundle/asset-properties.md

### To create an asset property {#to-create-an-asset-property}

> When you navigate to the **Asset properties** page following **Configuration > Asset properties** for the first time, there will be no asset properties present in the system except default properties. See [Default properties](#default-properties) to view the list of properties available.

Reviewer: ```suggestion
When you navigate to the **Asset properties** page following **Configuration > Asset properties** for the first time, there will be no asset properties present in the system except the default properties. See [Default properties](#default-properties) to view the list of properties available.
```

If the asset property is "Generator code" with the regular expression "^MCGEN[a-zA-Z0-9]*$", all generator codes must start with the prefix “MCGEN”.

> If any of the above restrictions are not selected, it is not applicable for the asset property during asset creation.

Reviewer: ```suggestion
If you do not select any of the restrictions above, it is not applicable for the asset property during asset creation.
```

#### Default value {#default-value}

> Assigns a default value to the asset property. If this field is left empty in the asset property and marked as required in the **Asset model**, you must enter a value during the asset creation.

Reviewer: ```suggestion
Assigns a default value to the asset property. If this field is left empty in the asset property and marked as required in the asset model, you must enter a value during the asset creation.
```

* **Minimum**:

> If selected, enter a number in the field on the right. When you create the asset, the asset property value cannot be lower or less than the **Minimum** provided.

Reviewer: @Vagdevi1998 why the bold formatting?

Author: Since Minimum is the name of the field.

Reviewer: ```suggestion
If selected, enter a number in the field **Minimum** on the right. When you create the asset, the asset property value cannot be lower or less than provided in the field.
```

{{< c8y-admon-info>}}

> If you entered a [default value](/dtm/asset-types/#default-value) beforehand, it must be greater than or equal to the **Minimum** provided. By default, this box is not selected.

Reviewer: ```suggestion
If you entered a [default value](/dtm/asset-types/#default-value) beforehand, it must be greater than or equal to the value provided in the field **Minimum.** By default, this box is not selected.
```

{{< /c8y-admon-info>}}

* **Default value**:

> Assigns a default value to the asset property. If this field is left empty in the asset property and marked as required in the **Asset model**, you must enter a value during the asset creation.

Reviewer: ```suggestion
Assigns a default value to the asset property. If this field is left empty in the asset property and marked as required in the asset model, you must enter a value during the asset creation.
```

To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. Without values for latitude and longitude, the marker is hidden. To see the marker, click anywhere on the map. Click or drag the marker to the preferred position to select the default value for latitude and longitude.

{{<c8y-admon-info>}}

> If a property with the label "Location" already exists, the default location property will be created with the label "Asset Location" and with the key "c8y_Position".<br>

Reviewer: ```suggestion
If a property with the label "Location" already exists, the default Location property will be created with the label "Asset Location" and with the key "c8y_Position".<br>
```
